config/toggles.js

```diff
@@ -96,6 +96,8 @@ const longTermToggles = {
     'unauthorized'),
   enableWorkerActivityLogs: envVarIncludes(pe, 'ENABLE_ACTIVITY_LOGS',
     'worker'),
+  enableJobCleanupActivityLogs: envVarIncludes(pe, 'ENABLE_ACTIVITY_LOGS',
+    'jobCleanUp'),
```

Comment:
should be 'jobCleanup' (since "cleanup" is a real unhyphenated word, the "u" does not get camel-cased)

Reply:
Done